torchsde/_core/base_solver.py

```diff
@@ -12,17 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+THRESHOLD = 1e-5  # Voodoo constant chosen based on results comparing adjoint gradient and analytical gradient.
```

Comment:
I know a man with the power... !

Reply:
Feel free to change this according to #33. But just by looking at #33, I'm not totally convinced that solution is fail free.

Reply:
Yeah, I expect this will be replaced by #33. I think #33 works!